_topic_maps/_topic_map.yml

```diff
@@ -107,6 +107,30 @@ Topics:
   File: admission-plug-ins
   Distros: openshift-enterprise,openshift-aro,openshift-origin
 ---
+Name: Disconnected environments
+Dir: disconnected
+Distros: openshift-enterprise,openshift-origin
+Topics:
+- Name: Updating a cluster in a disconnected environment
+  Dir: updating
+  Topics:
+    - Name: About cluster updates in a disconnected environment
+      File: index
+    - Name: Mirroring OpenShift Container Platform images
+      File: mirroring-image-repository
+    - Name: Updating a cluster in a disconnected environment using OSUS
```

Comment:
```suggestion
    - Name: Updating a cluster in a disconnected environment by using OSUS
```

Reply:
I would be okay with this but the actual title of the procedure is "Updating a cluster in a disconnected environment using the OpenShift Update Service", so the current naming more closely resembles the title of the page.

Also I guess technically the procedure is more like Updating a cluster in a disconnected environment _while_ using OSUS, so maybe I can consider renaming the title in a later PR